test/ed25519-test.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+var assert = require('assert');
+var fs = require('fs');
+var bn = require('bn.js');
+var hash = require('hash.js');
+var elliptic = require('../');
+var utils = elliptic.utils;
+var toArray = elliptic.utils.toArray;
+var eddsa = elliptic.eddsa;
+
+function toHex(arr) {
+  return elliptic.utils.toHex(arr).toUpperCase();
+}
+
+var MAX_PROGRAMMATIC = process.env.CI ? Infinity : 50;
```

Comment:
This is to allow the CI to run all the test vectors, but only 50 when running locally. Because it can be a bit of a pain to wait for them all.